test/echo-server.c

```diff
@@ -230,6 +230,15 @@ static int tcp4_echo_start(int port) {
     return 1;
   }
 
+#ifdef TCP_FASTOPEN
+  r = uv_tcp_fastopen(&tcpServer, 5);
+  if (r) {
+    /* TODO: Error codes */
```

Comment:
Are the TODOs intended to be fixed before merge?

Reply:
I guess, but I don't remember what I meant by this, it has been two years :)

Reply:
Ah, I see.  I am just mimicking the code around:
```
  r = uv_tcp_init(loop, &tcpServer);
  if (r) {
    /* TODO: Error codes */
    fprintf(stderr, "Socket creation error\n");
    return 1;
  }

  r = uv_tcp_bind(&tcpServer, (const struct sockaddr*) &addr, 0);
  if (r) {
    /* TODO: Error codes */
    fprintf(stderr, "Bind error\n");
    return 1;
  }
```